packages/rest/src/validation/request-body.validator.ts

```diff
@@ -137,7 +137,11 @@ function validateValueAgainstSchema(
     return;
   }
 
-  const validationErrors = validate.errors;
+  let validationErrors = validate.errors as AJV.ErrorObject[];
+
+  if (options.ajvTranslate) {
```

Comment:
Let's be little bit more defensive and check that `ajvTranslate` is actually a function. We don't want to see `TypeError: options.ajvTranslate is not a function`.

```suggestion
  if (typeof options.ajvTranslate === 'function') {
```